RELEASES.md

```diff
@@ -4,6 +4,7 @@ Version 0.8.10 (2021-XX-XX)
  * Škoda Karoq 2019 support thanks to jyoung8607!
  * Volkswagen Arteon 2021 support thanks to jyoung8607!
  * Volkswagen California 2021 support thanks to jyoung8607!
+ * Hyundai Santa Fe 2022 support thanks to sunnyhaibin!
```

Comment:
can you move this to the top? car ports are usually ordered alphabetically

Reply:
Done.